examples/using-babel-plugin-macros/pages/posts-custom/import-posts.macro.js

```diff
@@ -0,0 +1,58 @@
+// this is a more fine-tuned version of import-all.macro for our specific use-case.
```

Comment:
Files in the `pages/` directory are exported as pages, so we might want to move this to a different directory (`lib/` maybe ?)

Reply:
Ah, fair point!

Reply:
Actually no, I want `posts-custom` to be a page. This is good as-is 👍

Reply:
This file will generate `posts-custom/import-posts.macro` page, not `posts-custom` 😅

Reply:
Yep, every files under pages are considered a page